pkg/pod/status.go

```diff
@@ -420,6 +423,11 @@ func areStepsComplete(pod *corev1.Pod) bool {
 }
 
 func getFailureMessage(logger *zap.SugaredLogger, pod *corev1.Pod) string {
+	// If the pod was evicted return the eviction message.
```

Comment:
```suggestion
	// If a pod was evicted, use the pods status message before trying to determine
	// a failure message from the pod's container statuses.
	// A container may have a generic exit code that contains less information,
	// such as an exit code and message related to not being located.
```

Your PR description did a good job explaining why we're doing this instead of waiting for the fallthrough after the container status search. Let's include it here!

Reply:
sounds good to me, added